easybuild/easyconfigs/s/SentencePiece/SentencePiece-0.2.0-GCC-12.3.0.eb

```diff
@@ -8,9 +8,11 @@ description = "Unsupervised text tokenizer for Neural Network-based text generat
 github_account = 'google'
 
 toolchain = {'name': 'GCC', 'version': '12.3.0'}
+toolchainopts = {'debug': True}
```

Comment:
Did you really mean to keep this one?

Reply:
I did not (leftover from trying to figure out why external ABSL broke it so hard